news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,103 @@
+
+And lastly we attended an interesting talk about [Virus Total API](https://developers.virustotal.com/reference), an API used to cluster and classify potential malware on the web.
+
+
+# Breakout Sessions
+
+Breakout sessions were organised for the rest of the time, allowing participants to share ideas and experiences about many differents themes related to the summit.
+
+## Rapid Detection and prevention
+
+Content Management Systems are used by most websites in the world and the number of attacks keep increasing every year.
+
+It has been identified as a high priority for everyone to be able to quickly detect attacks and act as swiftly as possible accordingly.
+
+On PrestaShop's side, the open source project decided to greatly improve the communication about security issues and get closer to [YesWeHack](https://yeswehack.com/) to setup a Bug Bounty program.
+By doing so, we will detect faster more security issues, which will allow us to provide a very fast security fix following the criticality of the issue.
+
+## Automatic updates
+
+It is a well known fact that the main risk for a software is its age. Keeping it updated is mandatory as it will considerably reduce the risk of having security issues.
+
+Even if at a given time there are no known issues, nobody can predict the future. Something like a [0-day](https://en.wikipedia.org/wiki/Zero-day_(computing)) on PHP, Apache, Nginx could be found at any time. Keeping the software up-to-date also ensures it stays compatible with the latest security system softwares versions if you need to upgrade fast to patch a security issue.
+
+However, updating a CMS is riskier when you use a custom theme, incompatible modules, modules not compliant with best practices or if [BC Breaks](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#bc-breaks) are introduced in latest versions, due to security reasons or performance gain.
+
+We're not going to lie to each other, we've clearly been putting the autoupgrade module on the back, and that has impacted the PrestaShop project.
+Everyone knows how important and necessary it is for the community.
+
+During the summit, we spent a lot of time talking about the self upgrade process, and we will surely take time this year to make improvements for this topic!
+
+## Disclose + Distribute + Query security bulletins in a standardized way?
+
+At the beginning of the year, we had a major [security incident](https://build.prestashop.com/news/phpunit-security-issue-post-analysis/) which demonstrated that we need to improve our security process.
+
+For example, we wanted to report the incident as an official CVE. The root cause was similar to [CVE-2017-9841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9841) but slightly different with (with CGI/FastCGI context), so we tried to reach out to CVE authority [cve.mitre.org](https://cve.mitre.org/) in order to either update the entry or create a new one. We still have not received an answer.
```

Comment:
There was a mistake in the CGI/FastCGI part, let me know if this makes more sense

```suggestion
For example, we wanted to report the incident as an official [CVE](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures). The root cause was similar to [CVE-2017-9841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9841) but with a slightly different context (involving CGI/FastCGI). We tried to reach out to [CVE authority](https://cve.mitre.org/) in order to either update the entry or create a new one. However, as of June 2020, we have not received an answer yet.
```